modules/installation-osp-default-deployment.adoc

```diff
@@ -22,20 +22,19 @@ Your quota must meet the following requirements to run the {product-title} insta
 |Subnets               | 1
 |RAM                   | 112 GB
 |vCPUs                 | 28
-|Volume storage        | 175 GB
+|Volume storage        | 275 GB
 |Instances             | 7
 |Security groups       | 3
 |Security group rules  | 60
-|Swift containers      | 2
-|Swift objects         | 1
-|Swift available space | 10 MB or more
 |================================
 
-[NOTE]
-Swift space requirements vary depending on the size of the bootstrap Ignition file and image registry.
-
 A cluster might function with fewer than recommended resources, but its performance is not guaranteed.
 
+[IMPORTANT]
+====
+If OpenStack Object Storage (Swift) is available, it is used as the default backend for the {product-title} image registry. In this case, the volume storage requirement is 175GB. Swift space requirements vary depending on the size of the image registry.
```

Comment:
Probably we can say:

> If OpenStack Object Storage (Swift) is available _and operated by a user account with the `swiftoperator` role_, ...

Because the role is a hard requirement for Swift backend.

Reply:
@Fedosin Good point. Without the role, is Glance used or are things messier than that?

Reply:
@Fedosin ^^^ Adding this note, but question stands.